code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -2,94 +2,101 @@
 
 /datum/objective/crew/morgue //Ported from old Hippie
 	explanation_text = "Ensure the Medbay has been cleaned of any corpses when the shift ends."
-	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
+	jobs = list(
+		JOB_NAME_CHIEFMEDICALOFFICER,
+		JOB_NAME_GENETICIST,
+		JOB_NAME_MEDICALDOCTOR,
+	)
+	var/static/list/medical_areas = typecacheof(list(
+		/area/medical/cryo,
+		/area/medical/genetics/cloning,
+		/area/medical/exam_room,
+		/area/medical/medbay/aft,
+		/area/medical/medbay/central,
+		/area/medical/medbay/lobby,
+		/area/medical/patients_rooms,
+		/area/medical/sleeper,
+		/area/medical/storage
+	))
```

Comment:
Why not just get subtypes of /area/medical?

Reply:
Rooms like morgue are subtypes of medical.